frontend/src/container/NotificationWall.jsx

```diff
@@ -86,65 +89,167 @@ export class NotificationWall extends React.Component {
-          url = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
+    if (entityType === TLM_ENTITY.CONTENT) {
+      if (eventType === TLM_EVENT.created && contentType === TLM_SUB.COMMENT) {
+        return {
+          icon: 'comments-o',
+          text: props.t('{{author}} commented on {{content}} in {{workspace}}', i18nOpts),
+          url: `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.parentContentType}/${notification.content.parentId}`
         }
-        break
-      default:
-        icon = 'fa-bell'
-        text = `${notification.author} ${notification.type}`
-        url = notification.content
-          ? `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
-          : '/ui'
+      }
+
+      const url = `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
+
+      switch (eventType) {
+        case TLM_EVENT.CREATED: {
+          return {
+            icon: 'magic',
+            text: props.t('{{author}} created {{content}} in {{workspace}}', i18nOpts),
+            url: `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
+          }
+        }
+        case TLM_EVENT.MODIFIED: {
+          if (notification.content.currentRevisionType === 'status-update') {
+            return {
+              icon: 'random',
+              text: props.t('{{author}} changed the status of {{content}} in {{workspace}}', i18nOpts),
+              url: url
+            }
+          }
+
+          return {
+            icon: 'history',
+            text: props.t('{{author}} updated {{content}} in {{workspace}}', i18nOpts),
+            url: url
+          }
+        }
+        case TLM_EVENT.DELETED: {
+          return {
+            icon: 'magic',
+            text: props.t('{{author}} deleted {{content}} from {{workspace}}', i18nOpts),
+            url: `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
+          }
+        }
+        case TLM_EVENT.UNDELETED: {
+          return {
+            icon: 'magic',
+            text: props.t('{{author}} restored {{content}} in {{workspace}}', i18nOpts),
+            url: `/ui/workspaces/${notification.workspace.id}/contents/${notification.content.type}/${notification.content.id}`
+          }
+        }
+      }
+    }
+
+    if (entityType === TLM_ENTITY.SHAREDSPACE_MEMBER) {
+      switch (eventType) {
+        case TLM_EVENT.CREATED: return {
+          icon: 'user-o+plus',
+          text: props.user.userId === notification.user.userId
+            ? props.t('{{author}} added you to {{workspace}}', i18nOpts)
+            : props.t('{{author}} added {{user}} to {{workspace}}', i18nOpts),
+          url: `/ui/workspaces/${notification.workspace.id}/dashboard`
```

Comment:
I suggest that `/ui/workspaces/${notification.workspace.id}/dashboard` be a constant too, to follow the same style of code that you did for the type content.
```
const urlDashboard = `/ui/workspaces/${notification.workspace.id}/dashboard`
```